src/app/public/modules/dropdown/dropdown.component.ts

```diff
@@ -87,12 +114,27 @@ export class SkyDropdownComponent implements OnInit, OnDestroy {
     return this._label || this.getString('skyux_dropdown_context_menu_default_label');
   }
 
+  /**
+   * Provides an observable to send commands to the dropdown. The commands should respect
+   * [the `SkyDropdownMessage` type](https://developer.blackbaud.com/skyux/components/dropdown#skydropdownmessage-type).
+   */
+
   @Input()
   public messageStream = new Subject<SkyDropdownMessage>();
 
+  /**
+   * Specifies a title to display in a tooltip when users hover the mouse over the dropdown button. This property accepts `string` values.
+   */
   @Input()
   public title: string;
 
+  /**
+   * This property is deprecated. We recommend against using it. It specifies how users interact
```

Comment:
I would put the deprecated message underneath the normal description.
```
/**
 * Normal description here.
 * @deprecated We recommend against using it.
 */
```
The deprecation message might need to be reworded (up to @blackbaud-johnly), but that's the gist.